CHANGELOG.md

```diff
@@ -3,14 +3,248 @@
+- Update egui_tiles with fix for drag-and-drop-panic [#2555](https://github.com/rerun-io/rerun/pull/2555)
+- Convert objectron proto.py back to using typing.List [#2559](https://github.com/rerun-io/rerun/pull/2559)
+- Exclude from `objectron/proto/objectron/proto.py` from `just py-format` [#2562](https://github.com/rerun-io/rerun/pull/2562)
+- Fix pinhole visualization not working with camera extrinsics & intrinsics on the same path [#2568](https://github.com/rerun-io/rerun/pull/2568)
+- Fix: always auto-layout spaceviews until the user interveens [#2583](https://github.com/rerun-io/rerun/pull/2583)
+- Fix freeze/crash when logging large times [#2588](https://github.com/rerun-io/rerun/pull/2588)
+- Update egui_tiles to fix crash [#2598](https://github.com/rerun-io/rerun/pull/2598)
+- Fix clicking object with single instance (of every component) selecting instance instead of entity [#2573](https://github.com/rerun-io/rerun/pull/2573)
+- Cleanup internal data-structures when process has been forked [#2676](https://github.com/rerun-io/rerun/pull/2676)
+- Fix shutdown race-condition by introducing a flush_timeout before dropping data [#2821](https://github.com/rerun-io/rerun/pull/2821)
+- Fix ui-scale based point/line sizes incorrectly scaled when zooming based on horizontal dimension [#2805](https://github.com/rerun-io/rerun/pull/2805)
+- Fix visibility toggle for maximized Space Views [#2806](https://github.com/rerun-io/rerun/pull/2806)
+- Fix loading file via CLI [#2807](https://github.com/rerun-io/rerun/pull/2807)
+- Fix disconnected space APIs in Python SDK [#2832](https://github.com/rerun-io/rerun/pull/2832)
+- Avoid unwrap when generating authkey [#2804](https://github.com/rerun-io/rerun/pull/2804)
+
+#### 🧑‍🏫 Examples
+- Add example template [#2392](https://github.com/rerun-io/rerun/pull/2392)
+- Show hidden url search param in `app.rerun.io` [#2455](https://github.com/rerun-io/rerun/pull/2455)
+- Minimal example of running an intel realsense depth sensor live [#2541](https://github.com/rerun-io/rerun/pull/2541)
+- Add a simple example to display Open Photogrammetry Format datasets [#2512](https://github.com/rerun-io/rerun/pull/2512)
+- Move `examples/api_demo` -> `tests/test_api` [#2585](https://github.com/rerun-io/rerun/pull/2585)
+
+#### 📚 Docs
+- Docs: link to `rr.save` and suggest `rerun` instead of `python -m rerun` [#2586](https://github.com/rerun-io/rerun/pull/2586)
+- Update docs about transforms [#2496](https://github.com/rerun-io/rerun/pull/2496)
+- Fixup remaining usages of log_rigid3 in docs [#2831](https://github.com/rerun-io/rerun/pull/2831)
+
+#### 🎨 Renderer Improvements
+- Expose type erased draw data that can be consumed directly [#2300](https://github.com/rerun-io/rerun/pull/2300)
+- Use less `mut` when using `RenderContext` [#2312](https://github.com/rerun-io/rerun/pull/2312)
+
+#### ✨ Other Enhancement
+- introduce `re_tracing` [#2283](https://github.com/rerun-io/rerun/pull/2283)
```

Comment:
should go to dev-experience imho. removed the "enhancement" tag so it will be there on regenerating the log